server/fishtest/stats/stat_util.py

```diff
@@ -310,6 +311,44 @@ def update_SPRT(R, sprt):
     elif sprt["llr"] > sprt["upper_bound"] - o1:
         sprt["state"] = "accepted"
 
+def residuals(coeffs, table):
+    # If k1j,..,ktj is a row in a contingency table
+    # then we associate a statistic to the sum of ai*kij
+    # which is standard normal under the null hypothesis
+    # which states that the rows of the contingency table are
+    # obtained by sampling from the same distribution.
+    #
+    # Let Ki be the column sums, let nj be the sum of the
+    # kij and let N be the grand total. Then after conditioning
+    # on nj and K1,..,Kt, (k1j,...,ktj) follows a multivariate
+    # hypergeometric distribution. See
+    # https://en.wikipedia.org/wiki/Hypergeometric_distribution
```

Comment:
I'm confused by the reference to the hypergeometric distribution :(

Data in contingency table should follow:
 - poisson distribution if the marginal totals are not fixed by the sampling process
 - multinomial distribution if the grand total is fixed by the sampling process (NumGames test) or if one of the marginal total (in our case the games played by a worker) is fixed by the sampling process (this should be SPRT test analyzed by tasks) or we have explanatory variables (the workers) so we can conditioning on those marginal totals (poisson distribution conditioned on a total is a multinomial distribution, this should be SPRT test analyzed by workers)
 - hypergeometric distribution if both marginal totals are fixed by the sampling process

I mean: we can design a random test with the explanatory variables (workers) playing those numbers of games, but we can't design a random process with the workers having those results.